File: src/test/com/intellij/lang/jsgraphql/validation/GraphQLOperationsValidationTest.java
package com.intellij.lang.jsgraphql.validation;

import com.intellij.lang.jsgraphql.GraphQLTestCaseBase;
import org.jetbrains.annotations.NotNull;

public class GraphQLOperationsValidationTest extends GraphQLTestCaseBase {
    @Override
    protected @NotNull String getBasePath() {
        return "/validation/operations";
    }

    @Override
    protected void setUp() throws Exception {
        super.setUp();
        enableAllInspections();
    }

    public void testRepeatableDirectives() {
        doHighlightingTest();
    }

    public void testWrongTypes() {
        doHighlightingTest();
    }

    public void testDuplicates() {
        doHighlightingTest();
    }

}

File: tests/testSrc/com/intellij/lang/jsgraphql/validation/GraphQLOperationsValidationTest.kt
package com.intellij.lang.jsgraphql.validation

import com.intellij.lang.jsgraphql.GraphQLTestCaseBase
import com.intellij.openapi.progress.runBlockingCancellable

class GraphQLOperationsValidationTest : GraphQLTestCaseBase() {
  override fun getBasePath(): String {
    return "/validation/operations"
  }

  override fun setUp() {
    super.setUp()
    enableAllInspections()
  }

  fun testRepeatableDirectives() = runBlockingCancellable {
    doHighlightingTest()
  }

  fun testWrongTypes() = runBlockingCancellable {
    doHighlightingTest()
  }

  fun testDuplicates() = runBlockingCancellable {
    doHighlightingTest()
  }
}
